src/build/genprod.scala

```diff
@@ -180,9 +180,9 @@ class Function(val i: Int) extends Group("Function") with Arity {
  *  object Main extends App {%s}
  *  }}}
  *
- *  Note that `Function1` does not define a total function, as might
+ *  Note that `Function$i` does not define a total function, as might
```

Comment:
Thanks for the comment, removed it for `Function0` and `Function2`, and changed to the following for `Function1`:
*Note that the difference between `Function1` and [[scala.PartialFunction]] is that the latter can specify inputs which it will not handle.*